### PR TITLE
Move Uni Policy to main repo and adjust test

### PR DESCRIPTION
**Pull request by filmakarov**

### CODE REVIEW

File: contracts/external/policies/UniActionPolicy.sol [hunk 1]
```diff
@@ -76,7 +83,7 @@ contract UniActionPolicy is IActionPolicy {
     {
         require(status[id][msg.sender][account] == Status.Live);
         ActionConfig storage config = actionConfigs[id][msg.sender][account];
-        require(value <= config.valueLimit);
+        require(value <= config.valueLimitPerUse);
         ParamRule[16] memory rules = config.paramRules.rules;
```
Comment (zeroknots) [inline contracts/external/policies/UniActionPolicy.sol:87]: what does the [16] do here?

Comment (zeroknots) [inline contracts/external/policies/UniActionPolicy.sol:87]: could maybe use a storage  reference here?

Comment (filmakarov) [reply contracts/external/policies/UniActionPolicy.sol:87]: max amount of parameter rules per function. I think it's more than enough

Comment (filmakarov) [reply contracts/external/policies/UniActionPolicy.sol:87]: great catch, thank you! fixed

Comment (zeroknots) [reply contracts/external/policies/UniActionPolicy.sol]: but does this sload 16?

**Review by zeroknots — APPROVED** [review]

**Pull request merged**